Quadriga/src/main/java/edu/asu/spring/quadriga/domain/impl/workspace/TextFile.java

```diff
@@ -0,0 +1,63 @@
+package edu.asu.spring.quadriga.domain.impl.workspace;
+
+import org.springframework.stereotype.Component;
+
+import edu.asu.spring.quadriga.domain.workspace.ITextFile;
+
+@Component
+public class TextFile implements ITextFile {
+
+    private String fileName;
+    private String projectId;
+    private String workspaceId;
+    private String refId;
+    private String fileContent;
```

Comment:
This has nothing to do with networks. But a text is identified by a handle (ideally the text would be stored in a repository of some kind and there it would have a URI or handle that identifies it). Networks use that handle to refer to what texts they are created for. So, if we store texts locally, we need to know what is its handle that networks would use.